src/MLJDecisionTreeInterface.jl

```diff
@@ -138,7 +171,18 @@ function MMI.fit(m::RandomForestClassifier, verbosity::Int, X, y)
                              m.min_purity_increase;
                              rng=m.rng)
     cache  = nothing
-    report = NamedTuple()
+
+    if m.feature_importance != :none
+        fi = feature_importance_options[m.feature_importance](forest)
+        fi_pairs = collect(Dict(zip(features, fi)))
+        # sort descending
+        sort!(fi_pairs, by= x->-x[2])
+
+        report = (feature_importances=fi_pairs,)
+    else
+        report = NamedTuple()
+    end
+
```

Comment:
computation for feature importances should be put in `MMI.feature_importances` method. See comment above.